.github/actions/bump-submodule/action.yml

```diff
@@ -0,0 +1,35 @@
+name: 'Bump submodule'
+description: 'Bump specified submodule from third_party'
+
+inputs:
+  submodule:
+    description: Name of submodule to bump.
+    required: true
+    type: string
+  revision:
+    description: Git revision from submodule repository
+    required: true
+    type: string
+
+runs:
+  using: 'composite'
+  steps:
+    run: |
+      pushd third_party/${{ inputs.submodule }}
+      git fetch origin
+      git checkout ${{ inputs.revision }}
+      popd
+      # XXX: export the following environment variables to
+      # suppress git config warning. The commit is required to
+      # avoid resetting dirty changes in submodule.
+      export GIT_COMMITTER_NAME="integration testing workflow"
+      export GIT_COMMITTER_EMAIL="noreply@tarantool.org"
+      export GIT_AUTHOR_NAME="integration testing workflow"
+      export GIT_AUTHOR_EMAIL="noreply@tarantool.org"
```

Comment:
It looks like it can be moved to another section.

```yml
# as a global variable
env:
  MY_VAR: foo


jobs:
  my_job:
    env:
      # as a local for all steps
      MY_VAR: foo
    steps:
      - name: ...
        run: ...
        env:
          # as a local variable
          MY_VAR: foo
```
Using variables:
```yml
run: echo "$MY_VAR"
```

[Documentation](https://docs.github.com/en/actions/learn-github-actions/variables)

Reply:
Fixed, thanks!